src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/ConfirmationDialog.vue

```diff
@@ -28,7 +22,7 @@
         <v-card-actions>
           <v-row>
             <v-col class="text-right">
-              <v-btn rounded="lg" :class="{ 'mb-2': smAndDown }" variant="outlined" @click="cancel">{{ cancelButtonText }}</v-btn>
```

Comment:
I think this piece is needed to prevent the stacking of buttons when in mobile view. 

Current (without) 
![image](https://github.com/bcgov/ECC-ECER/assets/74216496/b1c70907-291c-4507-9a6a-ca3d66495c4e)

Reply:
good find, used https://vuetifyjs.com/en/styles/flex/#enabling-flexbox